src/publish.proj

```diff
@@ -24,4 +28,21 @@
     <!-- skip uploadToAzure when there are no nupkgs -->
     <CallTarget Targets="UploadToAzure" Condition="'@(_PackagesToPublish)' != ''" />
   </Target>  
+  
+  <Target Name="FinalPush" >
```

Comment:
Is this supposed to be `FinalPublish` like the `FinalPublishPattern`?  There seems to be no consistency between `Push` and `Publish`.  I'd prefer `Publish`, `push` sounds weird to me, but I'd tell you why offline.

Reply:
Continuing from https://github.com/dotnet/corefx/pull/24584#discussion_r144129505, I also prefer the verb Publish as it is consistent across all repos.